config/src/test/java/org/springframework/security/config/annotation/web/configurers/oauth2/client/OAuth2LoginConfigurerTests.java

```diff
@@ -367,6 +367,17 @@ public void oauth2LoginWithOneClientConfiguredThenRedirectForAuthorization() thr
 		assertThat(this.response.getRedirectedUrl()).matches("http://localhost/oauth2/authorization/google");
 	}
 
+	// gh-6802
+	@Test
+	public void oauth2LoginWithOneClientConfiguredAndFormLoginThenRedirectDefaultLoginPage() throws Exception {
```

Comment:
When I run this test, `DefaultLoginPageGeneratingFilter.formLoginEnabled` is `false`, even though it's enabled via `formLogin()` config.